configure.ac

```diff
@@ -261,9 +261,9 @@ AC_ARG_ENABLE([tiff],
 if test "x$enable_tiff" != "xno" -o "x$enable_tiff" = ""; then
     if test "x$TIFF_CFLAGS$TIFF_LIBS" = "x" ; then
         if $PKG_CONFIG libtiff; then
-            PKG_CHECK_MODULES([TIFF], [libtiff])
+            PKG_CHECK_MODULES([TIFF], [libtiff], [TIFF_PC=libtiff])
         else
-            PKG_CHECK_MODULES([TIFF], [libtiff-4])
+            PKG_CHECK_MODULES([TIFF], [libtiff-4], [TIFF_PC=libtiff-4])
```

Comment:
More '-4' version magic.

Reply:
Not at all: This is attempted after checking for module `libtiff`, and the matching name is recorded.

AFAIU `libtiff-4` is simply the name chosen by upstream TIFF.